### ISSUE TITLE
[Non-blocking] File not found on service start

### ISSUE THREAD

**Issue by remygrandin** [SEServerExtender/SEServerExtender#5]
I've noticed an error in the SESE logs when launching SESE as a service, here is the log relevant part : 

```
2015-03-24 15:09:27.5569|Info|SEModAPIInternal.API.Common.SandboxGameAssemblyWrapper.get_Instance|Finished loading SandboxGameAssemblyWrapper
2015-03-24 15:09:27.5739|Error|SEModAPIInternal.API.Server.ServerAssemblyWrapper.ReflectionUnitTest|System.IO.FileNotFoundException: Impossible de trouver le fichier 'C:\Windows\system32\SpaceEngineersDedicated.exe'.
Nom de fichier : 'C:\Windows\system32\SpaceEngineersDedicated.exe'
   à System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   à System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   à System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   à System.IO.File.InternalReadAllBytes(String path, Boolean checkHost)
   à SEModAPIInternal.API.Server.ServerAssemblyWrapper.get_InternalType()
   à SEModAPIInternal.API.Server.ServerAssemblyWrapper.ReflectionUnitTest()
2015-03-24 15:09:27.5739|Warn|SEModAPIExtensions.API.BasicUnitTestManager.RunBaseReflectionUnitTests|ServerAssemblyWrapper reflection validation failed!
```

However, the server is starting and working fine
SESE Version : 1.74.11.326

**Comment by dodexahedron** [comment]
You may need to specify gamepath as an argument, when launching as a service, to avoid this error.

**Comment by remygrandin** [comment]
I forgot to mention but I did pass the game path parameter at service start

**Comment by dodexahedron** [comment]
Hmmmm...
Obviously I must be clobbering the path you set, then, somewhere.
I'll see what I can find.

**Comment by dodexahedron** [comment]
Are you still getting this error?

**Comment by skulk** [comment]
This is not just a service issue, just try to start the SEServerExtender.exe while being in an other working directory/drive

**Comment by dodexahedron** [comment]
You can't launch from another working directory. The assemblies can't load properly if you do. You must be using the game folder as your working directory.

**Comment by remygrandin** [comment]
Fix for this proposed in pull request #32
